pegasus/sites.v3/code.org/public/educate/csc.md.partial

```diff
@@ -26,23 +39,35 @@ They're perfect for teachers who'd like to incorporate computer science into the
 
 {{ tutorial_promo2_single_course, heading: "", background_image: "/images/fit-1940/csc/poetry/tutorial_promo2_background_poetry.jpg", text_color: "white", grades: "3+", subject: "ELA", duration: "5 hours", box_background_color: "rgb(0,0,0,0.4)", right_heading: "Coding with Poetry", right_description: "Connect computer science and poetry in your classroom with this series of lessons.", right_specs: "", right_button_url: "https://studio.code.org/s/poetry-2021/?viewAs=Teacher", right_button_text: "View module" }}
 
-## Open-ended projects
+{{ csc_module, title: "Marine Life Ecosystems", image: "/images/csc/landing/module_marine_life.png", grades: "3-5", subject: "Science", duration: "4 hours", description: "Restore balance to a marine life ecosystem on the verge of overfishing.", button_text: "Learn more", url: "https://studio.code.org/s/csc-ecosystems-2022" }}
+{{ csc_module, title: "Star Quilts", grades: "3-5", image: "/images/csc/landing/module_star_quilts.png", subject: "Math", duration: "4 hours", description: "Design a Native American star quilt using rhombus patterns in The Artist.", button_text: "Learn more", url: "https://studio.code.org/s/csc-starquilts-2022" }}
```

Comment:
another tiny nit (sorry!): can we keep the properties in the same order? so `grades` after `image` here to be consistent with the others? I thought `grades` wasn't set on this one at first

Reply:
No need to apologize, I love it! Just updated.